README.md

```diff
@@ -1,157 +1,22 @@
 # omega-sentinel
 
-omega-sentinel is a chat bot built on the [Hubot][hubot] framework. It was initially generated by [generator-hubot][generator-hubot], and configured to be deployed on [Heroku][heroku] to get you up and running as quick as possible.
+omega-sentinel is a chat bot built on the [Hubot](https://hubot.github.com/) framework. For more info, see [HUBOT.md](HUBOT.md)
 
-This README is intended to help get you started. Definitely update and improve to talk about your own instance, how to use and deploy, what functionality he has, etc!
+## Development
 
-[heroku]: http://www.heroku.com
-[hubot]: http://hubot.github.com
-[generator-hubot]: https://github.com/github/generator-hubot
+1. Create/Update/Delete some scripts in the `scripts/` folder
+1. Run `./bin/hubot`
 
-### Running omega-sentinel Locally
+## Production
 
-You can test your hubot by running the following.
+Hubot is deployed using Docker and Docker Compose.
 
-You can start omega-sentinel locally by running:
+### Initial Setup
+1. Clone this repository
+1. Copy sample env file `cp .env.example .env`
+1. Modify the env file with secrets `$EDITOR .env`
+1. Build image `make build`
```

Comment:
If `build` is included in `up`, we don't need this step.

Reply:
Good call :+1: